### ISSUE TITLE
Call to a member function getGateway() on null

### ISSUE THREAD

**Issue by DieterHolvoet** [BitBagCommerce/SyliusMolliePlugin#127]
```
/vendor/bitbag/mollie-plugin/src/Action/ConvertPaymentAction.php in BitBag\SyliusMolliePlugin\Action\ConvertPaymentAction::execute at line 90
```

## Steps to reproduce
1. Configure multiple payment methods
2. Checkout an order and enter address/shipping
3. Choose a payment method powered by Mollie and click _Next_ instead of clicking on a specific payment method's button
4. In the next step, click on _Place order_
5. The page breaks

**Comment by xNordo** [comment]
Hi, on which version of SyliusMolliePlugin did this occured?

**Comment by DieterHolvoet** [comment]
v3.3.4

**Comment by xNordo** [comment]
This issue was already resolved in version 3.3.6, you may consider updating. Here's original issue https://github.com/BitBagCommerce/SyliusMolliePlugin/issues/75

**Comment by DieterHolvoet** [comment]
I'm sorry, my bad! That fixes the issue. Thank you for your help.
